src/clib/lib/enkf/model_config.cpp

```diff
@@ -426,7 +420,11 @@ void model_config_init(model_config_type *model_config,
     if (config_content_has_item(config, HISTORY_SOURCE_KEY)) {
         const char *history_source =
             config_content_iget(config, HISTORY_SOURCE_KEY, 0, 0);
-        source_type = history_get_source_type(history_source);
+        history_source_type source_type = REFCASE_HISTORY;
+        if (strcmp(history_source, "REFCASE_SIMULATED") == 0)
+            source_type = REFCASE_SIMULATED;
```

Comment:
Something is going on here as the source_type is never read.

Reply:
What do you mean? We read `history_source` from the configuration and then set `source_type` accordingly?

Reply:
I just noticed a compiler warning, but it might be a fluke.

Reply:
I think that might have been a fluke, resolving.